File: typescript/src/main/vsdk/toolkit/app/options/EnumDesc.ts
export class EnumDesc {
  public value: number;
  public name: string | null;
  public abbrev: number;

  public constructor();
  public constructor(value: number, name: string | null, abbrev: number);
  public constructor(value = 0, name: string | null = null, abbrev = 0) {
    this.value = value;
    this.name = name;
    this.abbrev = abbrev;
  }
}

File: java/testsuite/ApplicationCases/RenderparkApplication/src/main/java/vsdk/toolkit/app/options/EnumDesc.java
package vsdk.toolkit.app.options;

public class EnumDesc {
    public int value;
    public String name;
    public int abbrev;

    public EnumDesc() {
        value = 0;
        name = null;
        abbrev = 0;
    }

    public EnumDesc(int value, String name, int abbrev) {
        this.value = value;
        this.name = name;
        this.abbrev = abbrev;
    }
}
